crates/tauri-utils/src/config.rs

```diff
@@ -617,6 +617,11 @@ pub struct MacConfig {
   /// Translates to the bundle's CFBundleVersion property.
   #[serde(alias = "bundle-version")]
   pub bundle_version: Option<String>,
+  /// The name of the builder that built the bundle.
+  /// 
+  /// Translates to the bundle's CFBundleName property.
```

Comment:
Ah sorry, annotated the last file in my first comment. This one here should communicate the fallback as well.